services/app-web/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -15,7 +15,7 @@ const Crumb = (crumb: BreadcrumbItem) => {
     if (link) {
         return (
             <Breadcrumb>
-                <Link asCustom={NavLink} to={link}>
+                <Link asCustom={NavLink} to={link} end>
```

Comment:
This resolves an issue with [react router dom](https://reactrouter.com/en/main/components/nav-link#end) where the submission page was being read out as the current page due to the url on that page being contained in the `submissions/:submissionID/mccrs-record-number` url which resulted in "current page" being read out for both breadcrumbs